src/components/ex-page-list/ex-pagelist.scss

```diff
@@ -0,0 +1,178 @@
+// Defines the spacing between the page list items
+// Applied as margin right for each item that is not the last in its row
+$itemSpacing: 30px;
+
+.pagelist {
+  &-contentfragmentlist {
+    &__item {
+      @apply mt-component-spacing-small top-0 right-0;
+
+      &--list {
+        @apply w-full;
+      }
+    }
+
+    @screen sm {
+      @apply flex flex-wrap;
+
+      &.one-column {
+        .pagelist-contentfragmentlist {
+          &__item {
+            @apply w-full;
+          }
+        }
+      }
+
+      &.two-column,
+      &.four-column {
+        .pagelist-contentfragmentlist__item {
+          @apply w-table-1/2;
+          margin-right: $itemSpacing;
+
+          &:nth-of-type(2n) {
+            @apply mr-0;
+          }
+        }
+      }
+
+      &.three-column {
+        .pagelist-contentfragmentlist__item {
+          @apply w-table-1/3;
+          margin-right: $itemSpacing;
+
+          &:nth-of-type(3n) {
+            @apply mr-0;
+          }
+        }
+      }
+    }
+
+    @screen md {
+      &.four-column {
+        .pagelist-contentfragmentlist__item {
+          @apply w-table-1/4;
+          margin-right: $itemSpacing;
+
+          &:nth-of-type(2n) {
+            margin-right: $itemSpacing;
+          }
+
+          &:nth-of-type(4n) {
+            @apply mr-0;
+          }
+
+          .pagelist-contentfragment {
+            &__title {
+              @apply text-xl;
+            }
+
+            &__meta-data {
+              @apply text-xs;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  &-contentfragment {
+    @apply h-full;
+    &__container {
+      @apply flex flex-col h-full p-0;
+    }
+
+    .cmp-image {
+      @apply h-full;
+    }
+
+    &__url {
+      @apply no-underline;
+
+      &:hover {
+        @apply no-underline;
+      }
+    }
+
+    &__image {
+      @apply mb-3;
+      img {
+        @apply object-cover w-full h-full;
+      }
+
+      @screen sm {
+        @apply mb-2;
+      }
+    }
+
+    &__meta-data {
+      @apply font-headline text-gray-0 tracking-default flex items-center mt-0 mb-1 text-xs font-light leading-none uppercase;
+
+      &:before {
+        @apply inline-block mr-2 bg-cover;
+
+        content: '';
+        width: 10px;
+        height: 10px;
+        background-image: url('../resources/images/sicherearbeit-arrow-right.svg');
```

Comment:
Same as in #6 : You could inject an HTML entity triangle or CSS shape instead of SVG.

Reply:
Changed to html arrow